### DISCUSSION TITLE
Y-axis squashed into bottom of screen

### DISCUSSION THREAD

**Discussion by Wils1907** [NightscoutFoundation/xDrip#3077] [Bug reports] [answered]
UPDATE:
Navid helped me fix this last time by deleting a flag (there was only one). Yesterday I had the same issue with a new sensor but there were no flags. There was a yellow dot at a very high value and deleting that fixed the axis this time. 

Hi

I started a new sensor yesterday and now the y-axis of the chart is compressed at the bottom of the screen.  

![image](https://github.com/NightscoutFoundation/xDrip/assets/89298613/693436fe-87a0-4260-a99e-9ebd928521e4)

I've force quit the app and restarted the phone and tried playing around with settings but it hasn't fixed it. 

Any suggestions greatly appreciated.

**Comment by Navid200** [comment, marked as answer]
Do you see these stop and start flags?
![266959143-693436fe-87a0-4260-a99e-9ebd928521e4](https://github.com/NightscoutFoundation/xDrip/assets/51497406/43dd3cca-f6a3-4468-b7e4-ed4f1c1ea156)

Tap on the red one.  Delete it.  That should reduce the compression.  
Then, do the same with the green one.  That should fix it.

**Comment by Navid200** [comment]
@Wils1907 
By the way, this happens occasionally and has been reported before.   We should fix it.
Please click on `Create issue from discussion` in the right pane if you want so that we will have a record of this as something to track.

**Reply by Wils1907** [reply]
Thanks very much, Navid200. There was a red flag but no green flag. The red one was so far into the top left corner of the chart that tapping on it didn't work. But then a reminder popped up from a different app and, when I dismissed it, the red flag had gone away and the chart had gone back to normal. So I'm not sure how, but it's fine now!

I've also noticed that for my last two sensors the message about gathering some readings before it can start working has disappeared. It still works OK, though. Is this a recent change? 

Thanks again...   Wils

**Reply by Navid200** [reply]
What is your hardware data source?

**Reply by Wils1907** [reply]
Sorry, I'm not sure what you mean. I use Libre 2 (UK version)  on a Samsung Galaxy S22+ Android phone running One UI v5.1, Android v13.
